docs/user-guide/configure-security-checklist.md

```diff
@@ -0,0 +1,59 @@
+# Configuring security checklist
+
+During the initial installation of Zowe server-side components, it is necessary for your organization's security administrator to perform a range of tasks that require elevated security permissions. As a security administrator, follow the procedures outlined in this checklist to configure Zowe and your z/OS system to run Zowe with z/OS.
+
+**Tip:** We recommend that you consult with your security administrator who has the necessary permissions to perform the following security configuration procedures. 
+
+## Validate and re-run `zwe init` commands
+
+**Important!** During installation, the system programmer customizes values in the zowe.yaml file. However, due to insufficient permissions of the system programmer, the `zwe init security` command fails without notification. 
```

Comment:
You are notified. 

You see what steps failed. But the result in the end is always succes even when there are errors. Weird behaviour but it is as it is.